index.html

```diff
@@ -281,16 +281,12 @@ <h1 class="title">About <br>Myself</h1>
         <section class="contact-me">
           <h1>Contact me</h1>
           <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
-
           <form>
-            <input type="text" name="username" placeholder="Enter your name" class="input"><br>
-            <input type="email" name="email" placeholder="Enter email address" class="input"><br>
-            <textarea name="message" id="" cols="30" rows="10" placeholder="Write your message here" class="input textarea"></textarea>
-
+            <input type="text" name="username" placeholder="Enter your name" class="input" maxlength="30" required><br>
+            <input type="email" name="email" placeholder="Enter email address" class="input" required><br>
+            <textarea name="message" id="" cols="30" rows="10" placeholder="Write your message here" class="input textarea" maxlength="500" required></textarea>
```

Comment:
- [ ] as part of the requirements, your form should submit data to [formspree service](https://formspree.io/html) , kindly add the `action` and `method` attributes to your form and implement the formspree link to receive data 🙏